crates/cairo-coverage/src/input/statement_category_filter.rs

```diff
@@ -10,6 +10,35 @@ use std::iter::once;
 use std::sync::LazyLock;
 
 pub static VIRTUAL_FILE_REGEX: LazyLock<Regex> = LazyLock::new(|| Regex::new(r"\[.*?]").unwrap());
+
+/// This is not the best way to do this, and I'm not proud of it.
```

Comment:
As I stated here, this is more of a workaround for the problem rather than a proper solution. However, diving into the compiler/VM to find a solution would likely take more time, and staking is scheduled to be deployed on Sunday, so I don't have much time to spare. I guess we need to stick with this approach for now. For the moment, I will release this as an RC until I am confident about it.
![sad-troll-face-why](https://github.com/user-attachments/assets/8736cf55-0f4a-4e57-b078-bb6ed479080d)

Reply:
https://github.com/software-mansion/cairo-coverage/issues/98